src/main/java/frc/robot/subsystems/scoring/ShooterIOSim.java

```diff
@@ -31,9 +39,15 @@ public void setKickerVolts(double volts) {
     public void updateInputs(ShooterIOInputs inputs) {
         shooterSim.update(Constants.loopTime);
 
-        shooterAppliedVolts = shooterFeedforward.calculate(shooterSim.getAngularVelocityRadPerSec()) + shooterController.calculate(shooterSim.getAngularVelocityRadPerSec(), shooterGoalVelRPM * 2.0 * Math.PI / 60.0);
+        shooterAppliedVolts =
+                shooterFeedforward.calculate(shooterSim.getAngularVelocityRadPerSec())
+                        + shooterController.calculate(
+                                shooterSim.getAngularVelocityRadPerSec(),
+                                shooterGoalVelRPM * 2.0 * Math.PI / 60.0);
+        shooterSim.setInputVoltage(shooterAppliedVolts);
 
-        inputs.shooterVelocityRPM = shooterSim.getAngularVelocityRadPerSec() * 60.0 / (2.0 * Math.PI);
+        inputs.shooterVelocityRPM =
+                shooterSim.getAngularVelocityRadPerSec() * 60.0 / (2.0 * Math.PI);
```

Comment:
Could we turn 60.0 / 2pi into a Constants called `kRadPerSecToRPM` and then use it everywhere it is relevant? That would be helpful just for future readability of units